lightning/src/routing/network_graph.rs

```diff
@@ -1857,8 +1940,26 @@ mod tests {
 			};
 		}
 
-		// Permanent closing deletes a channel
-		{
+		if remove_by_timeout {
+			network_graph.remove_stale_channels_with_time(100 + 60 * 60 * 24 * 14);
+			assert_eq!(network_graph.read_only().channels().len(), 1);
+			assert_eq!(network_graph.read_only().nodes().len(), 2);
+
+			network_graph.remove_stale_channels_with_time(101 + 60 * 60 * 24 * 14);
```

Comment:
Not sure I understand what is going on here with `100` vs `101`. Aren't both well in the past?

Reply:
The updates created in the test have timestamps of 100.